tests/system/large/test_streaming.py

```diff
@@ -14,10 +14,13 @@
 
 import time
 
+import pytest
+
 import bigframes
 import bigframes.streaming
 
 
+@pytest.mark.flaky(retries=2, delay=120)
```

Comment:
I think we don't need to apply delay. Or 10s could be enough unless the service is having a large outage.

Reply:
done, along with retry=3